kube/services.go

```
@@ -15,3 +19,22 @@ func (c *Client) CreateService(service *v1.Service) error {
 
 	return err
 }
+
+// UpdateService will update an existing service
+func (c *Client) UpdateService(service *v1.Service) error {
+	s, err := c.k8sClient.Services(service.Namespace).Get(service.Name)
+
+	if err != nil {
+		fmt.Println(err)
+	}
```

This should return the error and not continue, instead of just printing